tests/test_patch_rebaser.py

```diff
@@ -113,7 +114,7 @@ def test_update_remote_patches_branch_without_dev_mode(mock_repo):
     assert mock_repo.tag.delete.called is False
     assert mock_repo.git.push.called is True
 
-    expected = [(("my_remote", "private-rebaser-2019-previous"),),
+    expected = [(("my_remote", "private-rebaser-unkown-2019-previous"),),
```

Comment:
Should this be 'unknown'?

Reply:
Whooops! I was worried it meant the test wasn't working but it looks like I consistently made the same mistake on L251 too. Will fix, thank you!!

Reply:
Done